pages/linux/gopass.md

```diff
@@ -0,0 +1,35 @@
+# gopass
+
+> Standard Unix Password Manager for Teams. Written in Go.
+
+- Startup config:
+
+`gopass init`
+
+- Show all "Keystores":
+
+`gopass mounts`
+
+- Mount a shared git "keystore":
+
+`gopass mounts add keystore_name location_git_repo`
+
+- Search interactive using a keyword:
+
+`gopass show keyword`
+
+- Search using a keyword:
+
+`gopass find keyword`
+
+- Sync all mounted "keystores":
+
+`gopass sync`
+
+- Show particular password entry:
```

Comment:
Show this entry based on what? What is used to do the search? 
I think it would be good to clarify that on the description.

Reply:
This command shows one entry in a specific location. That location if composed by keystore_name/..../entry.
Between the keystore and the entry, you can have tons of sub-folders.

Reply:
oh, understood. seems fine the way it is then.

Reply:
:+1: